MAUI/TreeMap/getting-started.md

```diff
@@ -267,6 +267,8 @@ public class PopulationViewModel
 {% endhighlight %}
 {% endtabs %}
 
+![getting-started-with-maui-tree-map](images/getting-started/getting-started-with-maui-tree-map.png)
+
```

Comment:
Add note for legend, levels, brush setting - to get more details on by navigating the respective md file link.